parity/main.rs

```diff
@@ -208,6 +208,22 @@ fn latest_exe_path() -> Option<PathBuf> {
 		.and_then(|mut f| { let mut exe = String::new(); f.read_to_string(&mut exe).ok().map(|_| updates_path(&exe)) })
 }
 
+fn set_spec_name_override(spec_name: String) {
+	if let Err(e) = File::create(updates_path("spec_name_overide"))
```

Comment:
Why is there a need for a file at all? Can't the hypervisor just pass CLI params to a new process?